apps/sync/doc/example-adapter.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/* exported ExampleAdapter */
+
+var ExampleAdapter = {
+  update: function(kintoCollection) {
+    kintoCollection.list().then(list => {
+      for(var i=0; i<list.data.length; i++) {
```

Comment:
I know this is an example only, but let's keep the same coding style everywhere:
`for (var i = 0; i = list.data.length; i++)`